code/datums/uplink_item.dm

```diff
@@ -344,6 +351,13 @@ var/list/uplink_items = list()
 	cost = 12
 	job = list("Station Engineer", "Chief Engineer")
 
+/datum/uplink_item/jobspecific/syndietape_engineering
+	name = "Syndicate Tape"
+	desc = "A length of engineering tape charged with a powerful electric potential. Will spark and shock people who attempt to remove it, creating fires. Can be used 3 times."
+	item = /obj/item/taperoll/syndie/engineering
+	cost = 4
+	job = list("Security Officer", "Warden", "Head of Security")
```

Comment:
🤔

Reply:
The lore reason for why they carry a toolbox of tools in their backpack of course